packages/core/src/BlockNoteEditor.ts

```diff
@@ -70,7 +55,7 @@ export type BlockNoteEditorOptions<BSchema extends BlockSchema> = {
    *
    * @default defaultSlashMenuItems from `./extensions/SlashMenu`
    */
-  slashCommands: BaseSlashMenuItem<any>[];
+  slashMenuItems: BaseSlashMenuItem<any>[];
```

Comment:
For this breaking change, we should:

a) check in the constructor whether `(options as any).slashCommands` has been passed, and log an error to explain this has been renamed
b) search the project for `slashcommands`, I think there might be some unrenamed values
c) make sure to mention it in changelog